icu4c/source/i18n/number_longnames.cpp

```diff
@@ -364,7 +561,7 @@ UnicodeString LongNameHandler::getUnitPattern(
         return ICU_Utility::makeBogusString();
```

Comment:
why there is no getUnitPattern in Java. ?

Reply:
I see there's quite a big difference in MeasureFormat.formatMeasure implementations - which is the only place where getUnitPattern is used. MeasureFormat is old / not recommended, so I think you can ignore its absence.

I documented the C++ one to stop myself from worrying about it. :-)

Reply:
+1